charter.html

```diff
@@ -160,22 +171,20 @@ <h1 id="title">PROPOSED Private Advertising Technology Working Group Charter</h1
       <section id="scope" class="scope">
         <h2>Scope</h2>
         <p>
-            The Working Group will specify new web platform features intended to
-            be implemented in browsers or similar user agents. The purpose of
-            these features is to support web advertising and provide users with
-            privacy guarantees with a strong technical basis.
+          The Working Group will specify solutions and essential inputs. The purpose of these solutions is to support
+          web advertising and user's Privacy.
         </p>
         <p>
-          The Working Group may consider designs that allow user agents for the
-          same user — including non-browser agents, like Operating Systems — to
-          collaborate in providing advertising features.
-          </p>
+          The Working Group may consider solutions that allow user's to receive a seamless experience across all the
+          devices they use.
         </p>
+      </section>
 
-        <section id="section-out-of-scope">
-          <h3 id="out-of-scope">Out of Scope</h3>
-          <p>Features that support advertising but provide privacy by means that are primarily non-technical should be proposed elsewhere.</p>
-        </section>
+      <section id="section-out-of-scope">
+        <h3 id="out-of-scope">Out of Scope</h3>
+        <p>
+          Features and solutions that add new web browser APIs specifically for the purposes of web advertising.
```

Comment:
I find it concerning that this change has gone without comment.  This proposes a completely different remit for the working group relative to the community group.  I am opposed to this change.

Reply:
I think that we have not yet gotten to the point where this is readable as a standalone PR and comments up to this point were intended to focus the changes into something that was in a clear format that could be discussed. That said, I do find this change particularly baffling.

Reply:
@AramZS In what way is this baffling? Why should the W3C start a Working Group to create business sector specific APIs? What business sectors does the W3C choose to adopt this approach in? Insurance, travel, publishing, commerce, health, legal? 

The W3C must not be a vehicle for browser vendors to insert themselves into an ever increasing number of business sectors and models. If the group is to exist then it must focus on using general purpose APIs. If those general purpose APIs don't exist then it should propose them to the relevant groups. That is the intention of this change.

Reply:
With your clearer reply on the other issue, I've responded to this question there @jwrosewell https://github.com/patcg/meetings/issues/52#issuecomment-1130148024